.github/workflows/app-build-docker-image.yml

```
@@ -91,7 +122,9 @@ jobs:
             APP_VERSION=${{ inputs.application-version }}
             BUILD_SHA=${{ github.sha }}
             ${{ inputs.build-args }}
-          tags: ${{ needs.determine-image-name.outputs.docker-image-name-with-tag }}
+          tags: |
+            ${{ needs.determine-image-name.outputs.gchr-docker-image-name-with-tag }}
+            ${{ needs.determine-image-name.outputs.dockerhub-image-name-with-tag }}
```

If following the previous proposal to have a single image-name-with-tag, you can either do the prepend here like the following or at the point where the tags are created if you think it is clearer to have separate outputs for the tags (_as in the suggestion for job `determine-image-name`_).

```suggestion
          tags: |
            "ghcr.io/${{ needs.determine-image-name.outputs.docker-image-name-with-tag }}"
            ${{ needs.determine-image-name.outputs.docker-image-name-with-tag }}
```

---

This looks great, much simpler than my approach